src/app/d2-loadout-builder/LoadoutBuilderItem.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import { D2Item } from '../inventory/item-types';
+import ConnectedInventoryItem from '../inventory/ConnectedInventoryItem';
+import { LockedItemType } from './types';
+import ItemPopupTrigger from '../inventory/ItemPopupTrigger';
+import DraggableInventoryItem from '../inventory/DraggableInventoryItem';
+
+export default function LoadoutBuilderItem({
+  item,
+  locked,
+  onExclude
+}: {
+  item: D2Item;
+  locked?: LockedItemType[];
+  onExclude(item: LockedItemType): void;
+}) {
+  const handleShiftClick = (item) => {
+    onExclude({ type: 'exclude', item });
+  };
+
+  return (
+    <DraggableInventoryItem item={item}>
+      <ItemPopupTrigger item={item} onShiftClicked={handleShiftClick}>
+        <div
+          className={classNames({
+            'excluded-item': locked && locked.find((p) => p.item.index === item.index)
+          })}
+        >
```

Comment:
Yay composition!

Reply:
yea! - I was also wondering: how can I apply a class to any of those other tags so I don't need an additional div?

ex.
`<ConnectedInventoryItem className="foo" item={item} />`

Reply:
We can just pass className in as a prop, that’s a very common pattern.

Reply:
cool, makes sense. wasn't sure if there was something special